src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/aimbot/autobow/AutoBowAimbotFeature.kt

```diff
@@ -30,39 +30,45 @@ import net.ccbluex.liquidbounce.utils.aiming.RotationManager
 import net.ccbluex.liquidbounce.utils.aiming.RotationsConfigurable
 import net.ccbluex.liquidbounce.utils.aiming.data.Rotation
 import net.ccbluex.liquidbounce.utils.aiming.projectiles.SituationalProjectileAngleCalculator
+import net.ccbluex.liquidbounce.utils.aiming.projectiles.SituationalProjectileAngleConsiderMissCalculator
 import net.ccbluex.liquidbounce.utils.combat.TargetPriority
 import net.ccbluex.liquidbounce.utils.combat.TargetTracker
 import net.ccbluex.liquidbounce.utils.kotlin.Priority
 import net.ccbluex.liquidbounce.utils.render.OverlayTargetRenderer
 import net.ccbluex.liquidbounce.utils.render.trajectory.TrajectoryData
 import net.minecraft.item.BowItem
+import net.minecraft.item.CrossbowItem
 import net.minecraft.item.TridentItem
 
 /**
  * Automatically shoots with your bow when you aim correctly at an enemy or when the bow is fully charged.
  */
 object AutoBowAimbotFeature : ToggleableConfigurable(ModuleAutoBow, "BowAimbot", true) {
-
-    // Target
+    private val rotationConfigurable = RotationsConfigurable(this)
+    private val throughWall by boolean("ThroughWall", true)
```

Comment:
I would call it `ThroughWalls` since that is a bit more consistent with the usual naming.